script/GameObjects/Elements/Components/Capacitor.lua

```diff
@@ -101,6 +79,24 @@ function Capacitor:discharge(value)
     return undischargedAmount
 end
 
+function Capacitor:getHealth()
+    return self.healthCurr or 0.0
```

Comment:
i feel like this should be a general health component which can be added to different classes

Reply:
Rather than being a bug, this seems like a very reasonable enhancement to make at some point to all the Component code when we make a pass through all the code to improve it.

I agree with you that it make sense to do this refactoring. However, rather than doing it now when I am trying hard to get as much of the basic code in place as possible, would you mind writing up this suggested change as an Issue and marking it as a Code Improvement task to be done along with other refactoring/improvement opportunities?